[DOCS] Adds content to the 7.8 What's New doc (#68975)
elastic/kibana: docs/user/whats-new.asciidoc

[[whats-new]]
== What's new in {kib} {minor-version}
++++
<titleabbrev>What's new in {minor-version}</titleabbrev>
++++

Here are the highlights of what's new and improved in {minor-version}.
For detailed information about this release, check out the
and https://www.elastic.co/blog/kibana-7-8-0-released[release blog], {kibana-ref}/release-notes.html[release notes],
and {kibana-ref}/breaking-changes.html[breaking changes].

Other versions: {kibana-ref-all}/7.7/release-highlights-7.7.0.html[7.7] | {kibana-ref-all}/7.6/release-highlights-7.6.0.html[7.6] | {kibana-ref-all}/7.5/release-highlights-7.5.0.html[7.5] |
{kibana-ref-all}/7.4/release-highlights-7.4.0.html[7.4] | {kibana-ref-all}/7.3/release-highlights-7.3.0.html[7.3] | {kibana-ref-all}/7.2/release-highlights-7.2.0.html[7.2]
| {kibana-ref-all}/7.1/release-highlights-7.1.0.html[7.1] | {kibana-ref-all}/7.0/release-highlights-7.0.0.html[7.0]

coming[7.8.0]

//NOTE: The notable-highlights tagged regions are re-used in the
//Installation and Upgrade Guide

// tag::notable-highlights[]

[float]
[[navigation-7-8]]
=== Side navigation organized by use cases

> 

Reviewer: How about:

{kib} has a cleaner, more organized side navigation for the Elastic apps. By organizing the apps by use case, you can easily navigate to where you need to go. For example, when you're ready to work with your Logs, Metrics, APM, or Uptime data, you can quickly find where you need to go under *Observability*. The side navigation also includes *Recently viewed* to easily access previously opened apps.

{kib} has a cleaner, more organized navigation for guiding you
through the apps.  The new sidebar groups
solutions by category. For example, the *Observability* category
includes Logs, Metrics, APM, and Uptime. The sidebar also includes
*Recently viewed* to easily
access previous projects.

> Hidden by default, the sidebar is accessible

Reviewer: This might be a vegetarian thing, but I really don't like hamburger menu. Is this considered a standard term?

with a single click on the hamburger menu.  Use the *Dock Navigation* link
at the bottom of the sidebar to keep the navigation visible.

[role="screenshot"]
image:images/whats-new/7-8-navigation.png[Kibana side navigation organized by category]

[float]
[[dashboard-first-7-8]]

> === Dashboard first experience

Reviewer: `Dashboard first` or `Dashboard-first`?

custom navigation paths between dashboards. Drilldowns
work well when you have multiple dashboards that support different audiences
or provide different levels of analysis. For example, you might
have a dashboard that shows the overall status of multiple data centers,
and then create a drilldown that navigates to a dashboard for a single data center.
Drilldown paths are available to all users who view the dashboard.

[role="screenshot"]
image::drilldowns/images/drilldown_on_piechart.gif[Drilldown on pie chart that navigates to another dashboard]

[float]
[[dashboard-clone-7-8]]
=== Cloning of dashboard elements
{kib} now lets you {kibana-ref}/dashboard-create-new-dashboard.html#cloning-a-panel[create an identical copy] of
any panel on the dashboard.
You can modify the visualization in the new panel as needed, and then save your changes.
Cloning enables you to quickly view your data from a
new perspective.

[role="screenshot"]
image:images/clone_panel.gif[clone panel]

[float]
[[lens-7-8]]
=== New visualizations for Lens

{kibana-ref}/lens.html[Lens] introduces three new visualization types: pie chart, donut chart, and treemap.  New

> to {kib}, a treemap provides a hierarchical view of your data.

Reviewer: ```suggestion
to {kib}, treemaps provide a hierarchical view of your data and are good for visualizing part-to-whole relationships.
```

> Treemaps are good for visualizing part-to-whole relationships.

Reviewer: ```suggestion
```

[role="screenshot"]
image:images/whats-new/7-8-treemap.png[Treemap visualization]

[float]
[[canvas-7-8]]
=== Menu updates for Canvas

> {kibana-ref}/lens.html[Canvas] has a new top-level menu that makes it easier for you to find the actions you need.

Reviewer: ```suggestion
{kibana-ref}/lens.html[Canvas] has a new top-level menu that makes it easier for you to find the options to create, customize, and share your workpads.
```

> This menu provides access to charts and maps,

Reviewer: ```suggestion
```

> so you can quickly embed them in your workload.

Reviewer: ```suggestion
```

> The menu also includes actions for viewing, editing, and sharing your data.

Reviewer: ```suggestion
```

Reviewer: These options have always been there, they are now just easier to find with the new menu.

{kibana-ref}/pre-configured-action-types-and-connectors.html[preconfigured connectors], which
automatically start up with {kib}.
This capability enables you to pre-plumb alert connections
directly into Slack channels and designated email services.
Preconfigured connectors are available in every {kib} space, and require no additional
configuration after initialization.

[role="screenshot"]
image::user/alerting/images/pre-configured-connectors-managing.png[Connectors managing tab with pre-cofigured]

[float]
[[embed-swim-lanes-7-8]]
=== Embeddable machine learning swim lanes

In 7.8, you can view anomalies alongside your source data in a {kib} dashboard
with embedded swim lanes. From the *Add panel* menu, you can select the jobs of
interest, decide whether to view an overall score or split by job or influencer,
and choose the number of lanes to display.

[role="screenshot"]
image:images/whats-new/7-8-swim-lane.png[Embed swim lanes in dashboards]

[float]
[[file-upload-7-8]]
=== Increased limit for file upload

The upload limit for the File Data Visualizer increased from 100 MB to 1 GB,
so you can now upload larger CSV, JSON, and log files.
To configure this value, go to

> {kibana-ref}/advanced-options.html[Advanced Settings].

Reviewer 2: ```suggestion
{kibana-ref}/advanced-options.html#kibana-ml-settings[Advanced Settings].
```
